utility/RPi/RF24_arch_config.h

```diff
@@ -16,9 +16,11 @@
 
 #define _SPI spi
 
-#if defined SPI_HAS_TRANSACTION && !defined SPI_UART && !defined SOFTSPI
+#if defined (SPI_HAS_TRANSACTION)
```

Comment:
As this line seemed to have been straight copied from the RF24_config.h file, I removed some Arduino-only `#if defined` conditions to avoid any confusion.